styles.css

```diff
@@ -0,0 +1,65 @@
+* {
+    box-sizing: border-box;
+  }
+  
+  html {
+    
+      font-size: 10px;
+    }
+    
+  body {
+  
+      font-family: fantasy;
+      font-size: 2rem;
+      line-height: 1;
+      text-align: start;
+      color: black;
+      margin: 0;
+      background-color: gray;
+  
+    }
+    
+    h1 {
+      font-size: 4rem;
+      margin-bottom: 0;
+    }
+  
+    
+    #main {
+      margin: 30px 8px;
+      padding: 15px;
+      border-radius: 5px;
+      background: #eee;
+    }
```

Comment:
Well done using classes and id. I would recommend use responsive values in your CSS files. You could use values like ```margin: 15% 2%```

Reply:
> Well done using classes and id. I would recommend use responsive values in your CSS files. For example: `margin: 15% 2%;`

- [ ] I would suggest you to give reason why you suggest this changes this will help the student or developer to understand you points of view , for exanple :A responsive value enhances interaction, responsive value: help [meet accessibility standards](https://elementor.com/blog/wordpress-accessibility-elementor/). Relative units scale better on different devices because they can scale up and down according to another element’s size.